apps/aecore/src/aec_consensus_hc.erl

```diff
@@ -289,11 +289,11 @@ state_pre_transform_node(Type, Height, PrevNode, Trees) ->
                    {ok, Seed} = get_entropy_hash(Epoch + 2),
                    cache_validators_for_epoch({TxEnv, Trees}, Seed, Epoch + 2),
                    step_eoe(TxEnv, Trees1, Leader, Seed, 0);
```

Comment:
Sorry, I think now these three lines are misaligned (they were correct relative to `true` _before_ that was fixed).